metis-indexing/src/main/java/eu/europeana/indexing/solr/SolrDocumentPopulator.java

```diff
@@ -198,15 +198,15 @@ private void populateWithDateRanges(SolrInputDocument document, RdfWrapper rdfWr
     final ProxyType europeanaProxy = rdfWrapper.getProxies().stream().filter(RdfWrapper::isEuropeanaProxy).findFirst()
                                                .orElseThrow();
 
-    final List<String> proxyChoiceLinks = europeanaProxy.getChoiceList().stream()
+    final List<String> proxyChoiceLinks = europeanaProxy.getChoiceList()!=null? europeanaProxy.getChoiceList().stream()
```

Comment:
I cannot see why the choice list is empty here. Is this happening during indexing or tombstoning?

Reply:
It happened during indexing, I was receiving the message `unable to publish in solr server`. I reverted the change and tested again and it didn't happened anymore. done.